libs/net/vmspec.py

```
@@ -194,3 +198,63 @@ def wait_for_ifaces_status(
+            conditions=event["object"].status.conditions or [],
+            condition=condition,
+            status=status,
+            missing_or_false=missing_or_false,
+        ):
+            return
+
+    expected = f"{condition} absent or not {status}" if missing_or_false else f"{condition}={status}"
+    raise VMIConditionNotFoundError(f"VMI {vm.name}: {expected} was not reached within {timeout}s.")
+
+
+def _vmi_condition_met(conditions: list, condition: str, status: str, missing_or_false: bool) -> bool:
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)